src/Microsoft.DocAsCode.Metadata/ValidationResults.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Microsoft.DocAsCode.Metadata
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    public class ValidationResults
+    {
+        public ValidationResults(IEnumerable<ValidationResult> results)
+        {
+            Items.AddRange(from r in results where !r.IsSuccess select r);
+        }
+
+        public bool IsSuccess => Items.Count == 0;
+        public List<ValidationResult> Items { get; } = new List<ValidationResult>();
```

Comment:
'=>' will eval every time access the property, '=' will only eval in constructor.